README.MD

```diff
@@ -62,8 +62,9 @@ The below script is written in `Python 3.6`. Remember to choose the same in AWS
   - `globalVars['ReplicateAMI']` - If you want to use my _Serverless AMI Replicator_
 
 - `Copy` the code from `serverless-ami-backup.py` in this repo to the lambda function
-  - If you have a lot of Instances, then consider increasing the lambda run time, the default is `3` seconds.
- - `Save` the lambda function
+- `Increase` the lambda timeout acording to the number of instances and the size of each EBS volume. You could start with `60` seconds and then increasing or decreasing it based on your results, the default is `3` seconds.
```

Comment:
Minor Updates Required:
- `acording` to `according`
- `increasing or decreasing` to `increase or decrease`